### PR TITLE
acr: update to 1.9.3.

### PR DESCRIPTION
**Pull request by fnogcps**

### CODE REVIEW

File: srcpkgs/acr/template [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # Template file for 'acr'
 pkgname=acr
-version=1.7.2
+version=1.9.3
 revision=1
 archs=noarch
 build_style=gnu-configure
 short_desc="AutoConf Replacement"
-maintainer="criw <contact@thecriw.com>"
```
Comment (jnbr) [inline srcpkgs/acr/template:8]: Are you the current maintainer? Why is it orphaned?

Comment (fnogcps) [reply srcpkgs/acr/template:8]: Package has missed updates for almost 2 years, the maintainer hasn't been active

Comment (abenson) [reply srcpkgs/acr/template:8]: Considered adoption?

Comment (Piraty) [reply srcpkgs/acr/template:8]: @fnogcps mind adopting?

Comment (fnogcps) [reply srcpkgs/acr/template:8]: Yeah, no problem.

**Comment by fnogcps** [comment]
@Piraty Yeah, no problem.

**Pull request merged**